### PR TITLE
[feature/PI-489] Strip whitespace from incoming query params

### PR DESCRIPTION
**Pull request by jameslinnell**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: docker/sds-api/spine-directory-service/sds/request/tests/test_cpm_endpoints.py [hunk 1]
```diff
@@ -289,7 +289,23 @@ def test_allowed_query_params(self):
...
+            {
+                "org_code": "RTX ",
+                "interaction_id": "urn:nhs:names:services:ebs:PRSC_IN070000UK08",
+            },
+            {
+                "org_code": "RTX",
+                "interaction_id": " urn:nhs:names:services:ebs:PRSC_IN070000UK08",
+            },
+            {
+                "org_code": "RTX",
```
Comment (megan-bower4) [inline docker/sds-api/spine-directory-service/sds/request/tests/test_cpm_endpoints.py:306]: These are duplicated? Shouldn't they all be unique?

File: docker/sds-api/spine-directory-service/sds/request/tests/test_cpm_devices.py [hunk 2]
```diff
@@ -311,6 +311,22 @@ def test_allowed_query_params(self):
...
+            {
+                "org_code": "5NR ",
+                "interaction_id": "urn:nhs:names:services:lrs:MCCI_IN010000UK13",
+            },
+            {
+                "org_code": "5NR",
+                "interaction_id": " urn:nhs:names:services:lrs:MCCI_IN010000UK13",
+            },
+            {
+                "org_code": "5NR",
```
Comment (megan-bower4) [inline docker/sds-api/spine-directory-service/sds/request/tests/test_cpm_devices.py:327]: These are also duplicated?

File: docker/sds-api/spine-directory-service/sds/request/tests/test_cpm_endpoints.py (hunk 1 above)
Comment (jameslinnell) [reply docker/sds-api/spine-directory-service/sds/request/tests/test_cpm_endpoints.py:306]: It's testing accepted params, this is all, there are spaces inserted into each so they are not exact duplicates

Comment (megan-bower4) [reply docker/sds-api/spine-directory-service/sds/request/tests/test_cpm_endpoints.py:306]: Ah right! My skimming didn't see that 😅

**Pull request merged**